utilities/cassandra/cassandra_functional_test.cc

```diff
@@ -112,49 +112,51 @@ class CassandraFunctionalTest : public testing::Test {
     DB* db;
     Options options;
     options.create_if_missing = true;
-    options.merge_operator.reset(new CassandraValueMergeOperator());
+    options.merge_operator.reset(new CassandraValueMergeOperator(gc_grace_period_in_seconds_));
```

Comment:
Do we need to keep a reference of CassandraValueMergeOperator in case it get GCed?

Reply:
Option.merge_operator is shared_ptr and should be copied during the Open process, so we should be good.